gatsby/src/components/schema/schema.tsx

```diff
@@ -31,50 +39,100 @@ export const SchemaComp: React.FC<IProps> = ({
     {
       '@context': 'http://schema.org',
       '@type': 'WebSite',
+      url: 'https://covid.gov.cz',
+      name: langCode === 'en' ? 'Covid Portal' : 'Covid Portál',
```

Comment:
Jak tohle bude fungovat až přibudou další jazyky? V drupalu už máme nachystanou Vietnamštinu, Němčinu a Ruštinu a jak budou přeložený, tak postupně půjdou ven.

Reply:
Ano, o tom vím. Bude se to pak muset tahat z GraphQL, prozatím jsem nechal tohle řešení, které je ostatně pro zatím použito na více místech. Je ale jasné, že to bude chtít ještě dále poupravit, to stejné pak ještě hodnoty pro parametr `inLanguage`.